.github/workflows/wheels.yml

```diff
@@ -17,9 +17,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10"]
-        os: [ubuntu-20.04, macos-11.0]
+        python-version: [3.7, 3.8, 3.9]
+        os: [ubuntu-20.04, macos-10.15]
         platform: [x64]
+        include:
+          - os: ubuntu-20.04
+            python-version: "3.10"
+          - os: macos-11.0
```

Comment:
should we change this line too?

I think this is never run if `macos-11.0` is kept.

Reply:
Looks like it is being run, https://github.com/stan-dev/httpstan-wheels/actions/runs/1439510101

But I need to add "x64" to the new "include" lines.